examples/submit_job.py

```diff
@@ -0,0 +1,29 @@
+"""Example of how to submit a Dalton job using Python HTTP requests."""
+import requests
+import os
+
+DALTON_URL = os.getenv("DALTON_URL", "localhost")
+
+# this file needs to include the sensor configuration in yaml format. Make sure that the files are in the proper directory.
```

Comment:
You can use the API to dynamically populate this if you wanted to.

To get current Suricata or Snort sensors (https://github.com/secureworks/dalton/blob/master/app/dalton.py#L2328):

DALTON_URL/dalton/controller_api/get-current-sensors/suricata
DALTON_URL/dalton/controller_api/get-current-sensors/snort

Or to get all sensors (https://github.com/secureworks/dalton/blob/master/app/dalton.py#L2361):

DALTON_URL/dalton/controller_api/get-current-sensors-json-full

Then take the the `sensor_tech` or `tech` (for all sensors JSON) value and use that to pull the YAML (https://github.com/secureworks/dalton/blob/master/app/dalton.py#L418):

DALTON_URL/dalton/controller_api/request_engine_conf?sensor=<sensor_tech

Reply:
I added three API calls in the api client:

1. `get_current_sensors` implements API call to: DALTON_URL/dalton/controller_api/get-current-sensors-json-full
2. `get_sensor_tech`
  DALTON_URL/dalton/controller_api/get-current-sensors/suricata
  DALTON_URL/dalton/controller_api/get-current-sensors/snort
  
I am using these then to create the parameters for the `submit_job` API call